File: Day 2/arithmetic.c
#include <stdio.h>
#include <math.h>

int main()
{
	double originalPrice;
	int tax, tip;
	scanf("%lf %d %d", &originalPrice, &tip, &tax);
	double res = originalPrice + (double)originalPrice * tip / 100 + (double)originalPrice * tax / 100;
	int finalPrice = (int)round(res);
	printf("The final price of the meal is $%d.", finalPrice);
	return 0;
}

File: Day 2/arithmetic.js
function processData(input) {
    var originalPrice = parseFloat(input.split("\n")[0]);
    var tip = parseFloat(input.split("\n")[1]);
    var tax = parseFloat(input.split("\n")[2]);
    var res = originalPrice * (1 + tip / 100 + tax / 100);
    var finalPrice = Math.round(res);
    process.stdout.write("The final price of the meal is $" + finalPrice + ".");
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});
